incubator/sentry/values.yaml

```diff
@@ -0,0 +1,100 @@
+# Default values for sentry.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+image:
+  repository: sentry
+  tag: "8.17"
+  pullPolicy: IfNotPresent
+
+# How many web UI instances to run
+webReplicaCount: 1
+
+# How many cron instances to run
+cronReplicaCount: 1
+
+# How many worker instances to run
+workerReplicaCount: 2
+
+# Initial admin user to create 
+user:
+  email: admin@sentry.local
+  password: password
+
+# BYO Email server
+# TODO: Add exim4 template
+# https://docs.sentry.io/server/installation/docker/#outbound-email
+email:
+  from_address: sentry@sentry.local
+  host: smtp
+  port: 25
+  use_tls: false
+  user:
+  password:
+  enable_replies: true
+
+# Name of the service and what port to expose on the pod
+# Don't change these unless you know what you're doing
+service:
+  name: sentry
+  type: NodePort
+  externalPort: 9000
+  internalPort: 9000
+
+# Limits for the 
+resources:
+  limits:
+    cpu: 100m
+    memory: 128Mi
+  requests:
+    cpu: 100m
+    memory: 128Mi
```

Comment:
This limits are too low for sentry starting. I recommend at least 300m for cpu and 300Mi for memory.